zyte_common_items/pipelines.py

```diff
@@ -56,6 +61,11 @@ def process_item(self, item, spider):
         return ae.downgrade(item)
 
 
+class InfoDropItem(DropItem):
+    """DropItem subclass for items that should be dropped with an INFO message
+    (instead of the default WARNING message)."""
```

Comment:
If this class is defined here, we should define the log formatter here as well (not in zyte-spider-templates-project).

Reply:
Am I correct to assume that we should also enable the new formatter in the library add-on, and not in the project?

Reply:
Yes, it seems we should do it here.

Reply:
I meant the zyte-spider-templates library, given there is no add-on here. Or are you thinking we should add an add-on to zyte-common-items?

Reply:
You're adding it in https://github.com/zytedata/zyte-common-items/pull/124 already :)

Reply:
As for the settings, it seems it'd be better to keep them in the project - as we discussed elsewhere, it might be simpler for addons not to configure other libraries. But I don't have a stong opinion on this.